### PR TITLE
Add endpoint for stream configs

### PR DESCRIPTION
**Pull request by andchiind**

### CODE REVIEW

File: src/isar/apis/robot_control/robot_controller.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    @inject
+    def __init__(
+        self,
+        robot_utilities: RobotUtilities,
+    ):
+        self.robot_utilities: RobotUtilities = robot_utilities
+        self.logger = logging.getLogger("api")
+
+    def generate_media_stream_config(self):
+        return self.robot_utilities.generate_robot_media_config()
```
Comment (sondreo) [inline src/isar/apis/robot_control/robot_controller.py:25]: Consider unifying the naming of these two functions for simplicity.

File: src/isar/apis/robot_control/robot_controller.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    def get_info(self):
```
Comment (sondreo) [inline src/isar/apis/robot_control/robot_controller.py:27]: Is this function relevant for the robot controller?

File: src/isar/modules.py [hunk 3]
```diff
@@ -150,6 +161,13 @@ def provide_scheduling_utilities(
     ) -> SchedulingUtilities:
         return SchedulingUtilities(queues, mission_planner)
 
+class RobotUtilitiesModule(Module):
```
Comment (sondreo) [inline src/isar/modules.py:164]: As this new module is introduced, should we consider renaming the `UtilitiesModule` to `SchedulingUtilitiesModule` for specification?

File: src/isar/apis/robot_control/robot_controller.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        return RobotInfoResponse(
+            robot_package=settings.ROBOT_PACKAGE,
+            isar_id=settings.ISAR_ID,
+            robot_name=settings.ROBOT_NAME,
+            robot_map_name=settings.DEFAULT_MAP,
+            robot_capabilities=robot_settings.CAPABILITIES,
+        )
+
+    def _task_api_response(self, task: Task) -> TaskResponse:
```
Comment (sondreo) [inline src/isar/apis/robot_control/robot_controller.py:36]: Is this internal function used?

File: src/isar/services/utilities/robot_utilities.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+from isar.config.settings import settings
+
+from robot_interface.robot_interface import RobotInterface
+
+
+class RobotUtilities:
+    """
+    Contains utility functions for scheduling missions from the API. The class handles
+    required thread communication through queues to the state machine.
```
Comment (sondreo) [inline src/isar/services/utilities/robot_utilities.py:15]: Reword class description.

**Review by sondreo — CHANGES_REQUESTED** [review]
Great work on the new endpoint! See some suggestions and questions.

File: src/isar/apis/robot_control/robot_controller.py (hunk 2 above)
Comment (andchiind) [reply src/isar/apis/robot_control/robot_controller.py:27]: I think so, since it gives information about the robot which is unrelated to missions

**Comment by andchiind** [comment]
Migrated to new branch https://github.com.mcas.ms/equinor/isar/pull/651

**Pull request closed without merging**